### PR TITLE
Travel fund request for Tiriel - Spring Collab Summit

### PR DESCRIPTION
**Pull request by Tiriel**
Hi everyone!

## Details:
 * **Event:** Spring 2018 Collaboration Summit
 * **Event Location:** Berlin, Germany
 * **Travelling from:** Paris, France
 * **Purpose:** Talks and facilitation for CommComm, Inclusivity Initiative, & i18n WG
 * **Stipend:** 400$ (max, provisional, will probably be more like 300$)
 * **Foundation member:** Of course
 * **Corporate travel fund:** Sadly not

## Stipend details:
 * **Flight:** 150$
 * **Hotel:** 250$

## Tags:
@nodejs/community-committee , @nodejs/i18n 

I must note this would be my first ever conf/event, I'd love if this could be done and meet everyone!
Cheers!

### CODE REVIEW

**Comment by Tiriel** [comment]
@MylesBorins I have! Since 25/10/17 in fact. Although, if I have a member ID, I can't find it. But I do have my order number if needed.

**Comment by Tiriel** [comment]
Quick question, one thing isn't really clearly stated in the doc. Do I have to wait for a quorum to be reached before I can consider the request is accepted? Or a certain time (one month is mentioned, but I'm not sure)?

Obviously, I'd like to make the reservations ASAP to avoid prices getting too high, but I really can't pay for it myself if I'm not sure I'll have it funded.

**Comment by obensource** [comment]
I too am curious about the answer to @Tiriel's [question](https://github.com/nodejs/admin/pull/86#issuecomment-372488362).^

When is it safe to purchase the tickets & reservations? If the prices have risen between the time that we PR'd the total cost and had it approved by the Foundation, is there repayment wiggle-room if the expense is (eg.) +$200 or so since? 

Thanks all! ❤️

**Comment by hackygolucky** [comment]
@Tiriel @obensource It should be safe once it is approved and merged so long as you meet the requirements and provide the info. I've filed another issue making sure that approvers understand the responsibility they have when they are +1ing requests. The Travel Fund documentation is clear on the requester's side in terms of requirements: you must be an active Individual Member of the Node.js Foundation and provide the requested information to be approved.

It's the approver's side we need to improve documentation on! So I'll work on that.

**Comment by obensource** [comment]
Thank you @hackygolucky–really appreciate it (and you)! 😎

**Comment by Tiriel** [comment]
@hackygolucky Thanks for the information! So do we need to provide some kind of member id?
And what about the part mentioning to update the PR once the final stipend is known?

Again, thanks a lot!

**Comment by obensource** [comment]
@hackygolucky @Tiriel I'm wondering if the safest thing to do is to recalculate our totals every other day, and update our PRs accordingly.

**Comment by addaleax** [comment]
> If the prices have risen between the time that we PR'd the total cost and had it approved by the Foundation, is there repayment wiggle-room if the expense is (eg.) +$200 or so since?

I think in the past we have handled that on the basis that requesters provided an estimate that leaves a bit of wiggle room, and later changed the document (typically with another PR) which modifies the amount so that it matches the final costs.

**Comment by hackygolucky** [comment]
@Tiriel Will you resolved the conflicts on branch and then I'll merge!

**Comment by Tiriel** [comment]
@hackygolucky Thanks! I believe it is done

**Comment by Tiriel** [comment]
Hey!

I just made the bookings since this was going to be merged. To I update it with the final information or do I wait and open a new PR? (Final cost is a bit less than what I had provided)

File: .idea/admin.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
```
[the remark this answers is not in the document]
Comment (jakeNiemiec) [reply .idea/admin.iml]: @Tiriel, the same thing use to happen to me. 

Check out https://gist.github.com/subfuzion/db7f57fff2fb6998a16c for an example of how to globally ignore IDE config files.

[the remark this answers is not in the document]
Comment (Tiriel) [reply .idea/admin.iml]: 😱 oh my. So sorry! I usually take extra care about not committing those!

I’ll fix this as soon as I’m home. Thanks @JemBijoux @jakeNiemiec

[the remark this answers is not in the document]
Comment (Tiriel) [reply .idea/admin.iml]: Now we're good!

**Comment by Tiriel** [comment]
Here we go, updated with final stipends as sent to the Foundation.

PTAL (cc @JemBijoux  )

**Pull request merged**